packages/tools/pluggable-widgets-tools/configs/rollup-plugin-collect-dependencies.js

```diff
@@ -0,0 +1,155 @@
+import { basename, dirname, extname, join } from "path";
+import { copy, readJson, writeJson } from "fs-extra";
+import { promises } from "fs";
+import { rm } from "shelljs";
+
+export function collectDependencies(
+    externals,
+    outputDir,
+    shouldCopyNodeModules = true,
+    widgetName,
+    shouldRemoveNodeModules = true,
```

Comment:
Maybe call it `clean`?

Reply:
Also `shouldCopyNodeModules` and `shouldRemoveNodeModules` seems to be true together, so maybe they can be merged.

Reply:
JSActions should not remove always the node modules.. just in the first input

Reply:
Iiuc you can replace `shouldCopyNodeModules` and `shouldRemoveNodeModules` with one flag. Named e.g. `copyNodeModules`

Reply:
We cannot replace those 2 options with just one because for jsaction it can be:
(copy, remove)
true, false
true, true

For Widgets is:
true, true
false, true